### DISCUSSION TITLE
Binaries for Raspberry Pi 0 w

### DISCUSSION THREAD

**Discussion by pyIronManCoder** [coder/code-server#6185] [Q&A]
Hello All, First of all, thank you to all the contributors behind this project.
I needed some help installing code-server on raspberry pi 0 w, i.e. armv6I or armv6hf, If anybody has compiled this package of code-server please share it with me or if anybody knows a faster way to compile code-server for raspberry pi 0 w please do share.
Thank you in advance

**Comment by lg-nineoneninethreenine** [comment]
Well, I don't know if armv6l is supported here. You could test and tell by actually compiling code-server in the Zero W, using the guides in the documentation. Hope this helps.

**Reply by pyIronManCoder** [reply]
Hi thanks for quick reply, I spent 2 hours compiling for raspberry pi 0 w from the command "yarn global add code-server", It was a success after that I tried to run it by "code-server" command, it is showing "[2023-05-03T07:39:06.690Z] error timed out"  message, btw it isn't showing anything else except this. Still not able to figure out why it isn't working? Please help... Thanks

**Reply by lg-nineoneninethreenine** [reply]
Well, I don't really know how to fix this... Try asking @code-asher, let's hope you get your answer and that this will be fixed soon.

**Reply by pyIronManCoder** [reply]
Thanks, I also asked him on Slack and got reply sometime ago.
